benchmarks/postgres-kafka-snapshot/kafka-connect/init/init.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Postgres 16
+confluent-hub install --no-prompt debezium/debezium-connector-postgresql:2.5.4
```

Comment:
I found this out when I was working on the Dbz Mongo test: Confluent Hub doesn't have the latest version of the Dbz connectors, which is currently 3.1.

This is what I used to install the latest version: https://github.com/ConduitIO/streaming-benchmarks/pull/72/files#diff-20112f33774a89420ea8c6780fb16600d44a82be9eaf6c9d61c654507843550b (directly downloads the JAR from Maven).